frappe/public/js/frappe/form/controls/json.js

```diff
@@ -0,0 +1,29 @@
+frappe.ui.form.ControlJSON = frappe.ui.form.ControlData.extend({
```

Comment:
It's a better idea to extend `ControlCode` as it will support JSON syntax highlighting.

Reply:
Thanks, It's updated.

Reply:
By changing to ControlCode from ControlData it's raising a few issues If you can resolve this and change, please do.

Reply:
Updated. ControlJSON now extends ControlCode